README.md

```diff
@@ -1,19 +1,90 @@
 
-Workflows can be accessed within Warp directly within the app, either through the Command Palette or by pressing `ctrl-shift-r`.
+Workflows can be accessed directly within Warp, either through the Command Palette or by pressing `ctrl-shift-r`.
 
 All public workflows (i.e. workflows within this repo) are also available at [commands.dev](https://www.commands.dev/).
 
 ## Contributing
-
 Contributions are always welcome! If you have a workflow that would be useful to many Warp users, feel free to send a PR to add a Workflow spec.
 
-The existing preset files should serve as an example for the format, while the a more comprehensive description of the file format is available in [FORMAT.md](FORMAT.md). Please request a review from someone on the Warp team once a PR is ready for review.
+All workflows are defined as YAML files within the [`specs/`](specs/) directory. 
+
+### File Format
+A comprehensive description of the file format is available in [FORMAT.md](FORMAT.md).
+Additionally, see the workflow below as an example to quickly get started:
+
+```yaml
+---
+# The name of the workflow.
+name: Uninstall a Homebrew package and all of its dependencies
+# The corresponding command for the workflow. Any arguments should be surrounded with two curly braces. E.g `command {{arg}}`.
+command: "brew tap beeftornado/rmtree\nbrew rmtree {{package_name}}"
+# Any tags that workflow should be categorized with.
+tags:
+  - homebrew
+# A description the workflow.
+description: Uses the external command rmtree to remove a Homebrew package and all of its dependencies
+# List of arguments within the command.
+arguments:
+    # Name of the argument within the command. This must match exactly with the name
+    # of the argument within the command (without the curly braces).
+  - name: package_name
+    # The description of the argument.
+    description: The name of the package that should be removed
+    # The default value for the argument.
+    default_value: ~
+# The source URL for where the workflow was generated from, if any.
+source_url: "https://stackoverflow.com/questions/7323261/uninstall-remove-a-homebrew-package-including-all-its-dependencies"
+# The author of the workflow.
+author: Ory Band
+# The URL of original author of the Workflow. For example, if this workflow was generated from StackOverflow, the `author_url` would be the StackOverflow author's profile page.
+author_url: "https://stackoverflow.com/users/207894"
+# The valid shells where this workflow should be active. If valid for all shells, this can be left empty.
+shells: []
+```
+
+### Testing
+To test a workflow within Warp before submitting, you can use it as a local workflow within warp.
+
+To do this:
+1) Copy the workflow to your local `~/.warp/workflows` directory:
+    ```bash
+    mkdir -p ~/.warp/workflows && cp {{workflow}}.yaml; ~/.warp/workflows/
+    ```
+2) Open Warp and open workflows by pressing `ctrl-shift-r` or using the command palette.
```

Comment:
Should "workflows" be capitalized when we're talking about the UI component (e.g.: "Open Warp and open Workflows" or "Open Warp and open the Workflows panel")?

Reply:
from chatting with Zach I think we want to keep it lowercase